release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -284,6 +284,27 @@ For more information, see xref:../operators/operator_sdk/osdk-pkgman-to-bundle.a
 
 When running machines in vSphere, you can now set the `minReplicas` value to `0` in the `MachineAutoscaler` resource definition. When this value is set to `0`, the cluster autoscaler scales the machine set to and from zero depending on if the machines are in use. For more information, see the xref:../machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling[MachineAutoscaler resource definition].
 
+[id="ocp-4-8-mco-upgrade-complete"]
+==== MCO waits for all machine config pools to update before reporting the update is complete
+
+When updating, the Machine Config Operator (MCO) now waits for the worker machine config pools (MCP) to finish updating before reporting that the update is complete. Previously, the MCO reported update progress on the status of the master machine config pools, even if the worker pools were not done updating.   
```

Comment:
So I might be misremembering, but if I recall correctly the worker MCP doesn't block upgrade completing if it's not degraded. Is that the case @kikisdeliveryservice ? If so we should maybe clarify

Reply:
@yuqi-zhang Did I cover this in the following paragraph: _(MCO) now reports an `Upgradeable=False` status to the Cluster Version Operator (CVO) if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update._

Reply:
We need to delete that first paragraph, it's not correct and the 4.8 work is limited to only the second paragraph.

Reply:
@kikisdeliveryservice Done. Thanks!!